app/src/components/job/hr/HRJob.jsx

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
 import { getAllJobsAndSkills } from "src/api/jobs";
+import { BriefcaseIcon } from "@heroicons/react/20/solid";
 import JobTile from "../JobTile";
 
 function HRJob() {
```

Comment:
I'm wondering if this is the best way to structure the files because it seems that HRJob StaffJob ManagerJob all have lots of duplicative elements. Just something to think about but we can probably refactor it in refactor ticket next sprint.

Reply:
Maybe the base is the same, but currently the different roles are not differentiated well. i.e. HRJob as it is now has extra buttons, and sees both active and inactive roles. StaffJob should only be displaying active roles (by calling an altogether different endpoint) but it is not the case currently. ManagerJob hasn't been well defined yet so I can't comment on that